apps/web/src/server/applications/case/representations/representation/check-answers/__tests__/check-answers.test.js

```diff
@@ -0,0 +1,71 @@
+import { parseHtml } from '@pins/platform';
+import nock from 'nock';
+import supertest from 'supertest';
+import { createTestEnvironment } from '../../../../../../../../testing/index.js';
+import { representationFixture } from '../../../__fixtures__/representations.fixture.js';
+
+const { app, installMockApi, teardown } = createTestEnvironment();
+const request = supertest(app);
+
+const mockCaseReference = { title: 'mock title', status: 'in test', reference: 'mock reference' };
+const nocks = () => {
+	nock('http://test/').get('/applications/1').reply(200, mockCaseReference);
+	nock('http://test/').get('/applications/1/representations/1').reply(200, representationFixture);
+};
+
+describe('Representation address details page', () => {
+	beforeEach(installMockApi);
+	afterEach(teardown);
+
+	afterAll(() => {
+		nock.cleanAll();
+	});
+
+	const baseUrl = '/applications-service/case/1/relevant-representations/check-answers?repId=1';
+
+	describe('GET /applications-service/case/1/relevant-representations/check-answers', () => {
+		beforeEach(async () => {
+			nocks();
+		});
+
+		it('should render the page and have the lookup fields', async () => {
+			const response = await request.get(baseUrl);
+			const element = parseHtml(response.text);
+
+			expect(element.innerHTML).toMatchSnapshot();
+			expect(element.innerHTML).toContain('Organisation name');
+			expect(element.innerHTML).toContain('Full name');
+			expect(element.innerHTML).toContain('Job title');
+			expect(element.innerHTML).toContain('Email address');
+			expect(element.innerHTML).toContain('Address');
+			expect(element.innerHTML).toContain('Preferred contact method');
+			expect(element.innerHTML).toContain('Type');
+			expect(element.innerHTML).toContain('Under 18');
+			expect(element.innerHTML).toContain('Representation on behalf of');
+			expect(element.innerHTML).toContain('Agent details');
+			expect(element.innerHTML).toContain('Representation');
+			expect(element.innerHTML).toContain('Date received');
+			expect(element.innerHTML).toContain('Original representation');
+		});
+	});
+
+	describe('POST /applications-service/case/1/relevant-representations/check-answers', () => {
+		beforeEach(async () => {
+			nocks();
+			nock('http://test/')
+				.patch(`/applications/1/representations/1`, {
+					status: 'AWAITING_REVIEW'
+				})
+				.reply(200, { message: 'ok' });
+		});
+		describe('Field validation:', () => {
```

Comment:
Can you test the error states here for the whole page ? Just check the snapshot.